scripts/watch/client.sh

```diff
@@ -2,4 +2,4 @@
 
 cd client || exit -1
 
-npm run ng -- server --hmr --host localhost --port 3000
+npm run ng -- server --hmr --host 0.0.0.0 --port 3000
```

Comment:
This looks a bit scary, because you're exposing something to the world that was limited to `localhost` before. Could you please explain why this is needed? Is this a development server?

Reply:
It's not scary, it's only in development mode: note that this just means exposing the dev web server on the Docker network interface, which is needed if we want to expose the service outside of the container. Keeping it this way (the risk seems nonexistent to me).